cves/CVE-2002-0392.yml

```diff
@@ -199,4 +221,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This issue may have been mitigated if the system included additional unit test cases.
+    The test suite failed to include a case which evaluated this particular type of 
+    invalid input, and it brought about a vulnerability. The issue is that this particular
+    avenue of attack was never brought to mind and as such, the vulnerability passed
+    by unnoticed.
```

Comment:
Add a newline at the end of the file: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
On it!